org.sonatype.m2e.webby/src/org/sonatype/m2e/webby/internal/config/WarConfigurationExtractor.java

```diff
@@ -198,8 +198,7 @@ public WarConfiguration getConfiguration(IMavenProjectFacade mvnFacade, MavenPro
   private String resolve(String basedir, String path) {
     String result = path;
     if(path != null && basedir != null) {
-      path = path.replace('\\', File.separatorChar).replace('/', File.separatorChar);
-
+      path = path.replace('\\', '/');
```

Comment:
Who do you mean by "we"? Will this change work for other users of Webby?
What about Tomcat and Jetty, is this change compatible with them, especially on Windows?
AFAIK, "/" needs to be used in URLs, but File.separatorChar needs to be used in file system paths to be portable.